db/dcrpg/internal/blockstmts.go

```diff
@@ -198,7 +198,7 @@ const (
 	UpdateBlockNextByNextHash = `UPDATE block_chain SET next_hash = $2 WHERE next_hash = $1;`
 
 	// Grab the timestamp and chainwork.
-	SelectChainWork = `SELECT time, chainwork FROM blocks WHERE is_mainchain = true ORDER BY time;`
+	SelectChainWork = `SELECT time, chainwork FROM blocks WHERE is_mainchain = true ORDER BY height;`
```

Comment:
I actually wasn't certain under what conditions an ORDER BY column can be absent from the SELECT list, but PostgreSQL seems to be okay with it in this simple query.

Reply:
Unlike a group by or distinct, it's ok to order by anything even if not selected in the output.